tee-worker/enclave-runtime/src/stf_task_handler.rs

```diff
@@ -56,7 +56,12 @@ pub unsafe extern "C" fn run_stf_task_handler(dpc: *const u8, dpc_size: usize) -
 			dpc.set_sora_quiz_master_id(data_provider_config.sora_quiz_master_id);
 			dpc.set_sora_quiz_attendee_id(data_provider_config.sora_quiz_attendee_id);
 			dpc.set_nodereal_api_key(data_provider_config.nodereal_api_key);
+			dpc.set_nodereal_api_retry_delay(data_provider_config.nodereal_api_retry_delay);
+			dpc.set_nodereal_api_retry_times(data_provider_config.nodereal_api_retry_times);
```

Comment:
Shall we hardcode these numbers into enclave?

Reply:
The optimal value of these parameters won't be easily decided, and keeping it in config makes it can be tuned easily without needing to recompile

Reply:
I would opt for making these values same for all enclaves.
This retry strategy is still subject of change, see https://github.com/litentry/litentry-parachain/pull/2292/files#r1408881954